api/stream-portfolio/openapi.yaml

```diff
@@ -44,6 +47,37 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/Error'
+  ###############################################
+  ## Integration
+  ###############################################
+  /integration-api/v1/regions/batch:
```

Comment:
in the stream http services we are not adherent to the backbase standards of integration apis, I would rather keep it consistent to what we have today and create something like `/portfolios/regions/batch`